docs/getting-started/ingress-run.yaml

```diff
@@ -1,11 +1,12 @@
-apiVersion: tekton.dev/v1beta1
+apiVersion: triggers.tekton.dev/v1beta1
 kind: TaskRun
 metadata:
   name: create-ingress-run
   namespace: getting-started
 spec:
   taskRef:
     name: create-ingress
+  inputs:
```

Comment:
I think that since this is a v1beta1 resource, it doesn't have the `inputs:` line.

Reply:
good catch...we should start running these tests again.